package.json

```diff
@@ -0,0 +1,82 @@
+{
+    "name": "brite-rest",
+    "version": "0.0.0-semantically-released",
+    "description": "The official JavaScript SDK for the Eventbrite v3 API",
+    "author": "Eventbrite <npmjs@eventbrite.com>",
+    "main": "lib/umd/index.js",
```

Comment:
If they're using `main` it means they're using commonjs system, so it shoud just be lib/index.js, right?

Reply:
We have `lib/umd` (line 6) and `lib/esm` (lines 7 & 8). So I think we need the subfolders